### PR TITLE
Shard related code removed

### PR DESCRIPTION
**Pull request by motoras**
For details see [ticket](https://radixdlt.atlassian.net/browse/RPNV1-309)

### CODE REVIEW

File: src/test/java/com/radixdlt/identifiers/EUIDTest.java [hunk 1]
```diff
@@ -95,7 +95,6 @@ public void testGetLow() {
 	public void testGetShard() {
 		EUID euid = new EUID(UInt128.from(2L, 5L));
 		assertEquals(5L, euid.getLow());
-		assertEquals(2L, euid.getShard());
```
Comment (clstrfsck) [inline src/test/java/com/radixdlt/identifiers/EUIDTest.java:98]: This test is really a no-op now, and can be removed.  `getLow()` is tested in `testGetLow()`.

**Review by clstrfsck — COMMENTED** [review]
Please remove unit test that is no longer testing anything.

**Review by clstrfsck — CHANGES_REQUESTED** [review]
Unit test that doesn't test anything needs to be removed as noted.

File: src/test/java/com/radixdlt/identifiers/EUIDTest.java (hunk 1 above)
Comment (motoras) [reply src/test/java/com/radixdlt/identifiers/EUIDTest.java:98]: Done

**Review by clstrfsck — APPROVED** [review]
LGTM

**Pull request merged**